netjsonconfig/backends/raspbian/templates/commands.jinja2

```diff
@@ -0,0 +1,11 @@
+{% for i, j  in data.items() %}
+    {% if i|string() == 'general' %}
+        {% for general in j %}
+            {% if general.get('timezone') %}
+                run commands:
```

Comment:
Couldn't find a method to set timezone by modifying a config file. 
So created a template which tells what command needs to be run to do the required.